app/src/App.php

```diff
@@ -91,4 +91,17 @@ class App extends Kernel
         // fast code prototyping
         Prototype\PrototypeBootloader::class,
     ];
+
+    /**
+     * Get object from the container.
+     *
+     * @param string      $alias
+     * @param string|null $context
+     * @return mixed|object|null
+     * @throws \Throwable
+     */
+    public function get($alias, string $context = null)
```

Comment:
I disagree with such a change. Exposing direct access to a container outside of the application usually has no other purpose rather than testing or "dirty" access to needed instance (a.k.a. "Facade"). Keeping it in TestApp not only allows you to isolate test specific functionality but also configure custom bootload list, add your own interceptors and etc.

Reply:
I'll do some changes later.

Reply:
I have reverted changes to `App` and `TestApp`.